bot/helpers/db.py

```diff
@@ -0,0 +1,195 @@
+# db.py
+
+
+class DBHelper:
+    """ Class to contain database query wrapper functions. """
+
+    def __init__(self, pool):
+        """ Set attributes. """
+        self.pool = pool
+
+    @staticmethod
+    def _get_record_attrs(records, key='id'):
+        """ Get key list of attributes from list of Record objects. """
+        return list(map(lambda r: r[key], records))
+
+    async def _get_row(self, table, obj):
+        """ Generic method to get table row by object id. """
+        statement = (
+            f'SELECT * FROM {table}\n'
```

Comment:
Be very careful passing the table variable straight in like this, just ensure the user can't set this variable in any way or you'll be vulnerable to SQL injection

Reply:
I couldn't figure out how to pass in a query parameter for the table name. Probably worth double checking on that.